version/version.go

```diff
@@ -12,6 +12,6 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-package main
+package version
 
-const Version = "0.5.3+git"
+var Version = "0.5.3+was-not-built-properly"
```

Comment:
Fancy. I think this should be standardized across the company. I don't care what really gets included apart from the version but I do now like the idea of not using `sed` and generating those bump commits.